projects/ngx-tag-commander/src/lib/tag-commander.service/tag-commander.service.ts

```diff
@@ -48,15 +48,19 @@ export class TagCommanderService {
   //  * @param {string} id: the id the script node will have
   //  * @param {string} uri: the source of the script
   //  * @param {string} node: the node on witch the script will be placed, it can either be head or body
+  //  * @param {array} args: contains extra args for script tag like crossorigin
   // */
-  addContainer(id: string, uri: string, node: string): Promise<void> {
+  addContainer(id: string, uri: string, node: string, args: [string, string][]): Promise<void> {
```

Comment:
could we make the use of additional args optional so also undefined would be accepted, or empty array would be defaulted to, to provide backwards compatibility